frontend/consumer-ui/plugins/leaflet.js

```diff
@@ -1,5 +1,7 @@
 import Vue from 'vue'
 import { LMap, LMarker, LPopup, LTileLayer } from 'vue2-leaflet'
+// eslint-disable-next-line no-unused-vars
+import { GestureHandling } from 'leaflet-gesture-handling'
```

Comment:
Without that import here, the gesture handling doesn't happen. Only importing it like that without using it is enough to make that handling happen.